service/core-fuzzers.yaml

```diff
@@ -13,4 +12,3 @@ fuzzers:
     - lafintel
     - libfuzzer
     - mopt
-    - neuzz
```

Comment:
latest neuzz does not work on all benchmarks, see https://www.fuzzbench.com/reports/experimental/2021-02-08-paper/index.html, so removing from core-fuzzers for now.